sdks/python/setup.py

```diff
@@ -117,6 +117,7 @@ def get_version():
 REQUIRED_TEST_PACKAGES = [
     'nose>=1.3.7',
     'pyhamcrest>=1.9,<2.0',
+    'numpy>=1.14.3',
```

Comment:
This is a rather big dependency. Is it possible for us to not use it?

Reply:
Numpy is needed here because we do a zipf distribution (i.e. heavily weighted towards a few keys). Another option would be to have these files not be part of Beam, but since they will be used for perf tests, it'll likely be worth having them directly n beam.

Reply:
- Do we really need zipf? If we do can we find a light weight version?

If the answer is we still need numpy, I think using a new extra like `perftest` makes sense.